pkg/polarion-generator/test_cases_generator_test.go

```diff
@@ -50,30 +50,25 @@ var _ = Describe("Test Case Generator", func() {
 		return 1
 	}
 
-	table.DescribeTable("table test", func() {
-		// +polarion:caseposneg=positive
-		// +polarion:caseimportance=medium
+	table.FDescribeTable("[rfe_id:1, posneg:negative, vendor:cnv-qe.redhat.com, level:integration]table test", func() {
 		By("Testing the table")
 		testNameFunc("table")
 		testFunc1()
 
 	},
-		table.Entry("entry 1"),
-		table.Entry("entry 2"),
+		table.Entry("[test_id:1]entry 1"),
+		table.Entry("[component:Virt]entry 2"),
```

Comment:
For tests we must have test_id I guess. so table.Entry("**[component:Virt]**entry 2"), is valid?

Reply:
non of the test attr is a must, but some will fail the importer if they dont exist.
test if is not one of them, the importer will just assume its new test instead of a test which requires update